script/Arcueid/Day 5/05_00_ARC05_3.txt

```diff
@@ -58,24 +60,25 @@ After three days, my Lifescale will be back to around thirty percent of its orig
 [sha:108ca5be3f8903f4dfc82d856d66b63ac4b3c7db]{
 -- Page 221, Offset 9143.
 -- 「三日……？　いや、でも」
-"Three days...? But, that means―――"
+"Three days? But, that means―――"
 }
 [sha:d03ce8c4e257acf82f9ae01226f012b840fa917b]{
 -- Page 221, Offset 9144.
 -- 「ええ。その三日間、ヴローヴは野放しよ。アイツは地下に潜って人間から血液を補充する。……ううん、それだけじゃない。あの様子からすると夜の街を徘徊するかもしれない。
 "Yeah. For those three days, Vlov will go completely unchecked.
- He'll go into hiding and feed on humans in order to replenish his supply of blood... no, that's not all.
+ He'll go into hiding and feed on humans in order to replenish his supply of blood... on second though, he won't stop there.
  Judging by the state he's in, he might even roam the streets at night.
 }
 [sha:09cd56cf78f07e6251c23dd8e97d8c7daeaa4e8c]{
 -- Page 221, Offset 9145.
 -- アイツ、正気が欠けていたし。姿を隠さないと教会に狙われる、なんて<定石|セオリー>、頭にないのかもね」
-He's not in his right mind after all. I don't think he understands that the Church will come after him if he doesn't lie low."
+He's not in his right mind, after all. I don't think he understands that the Church will come after him if he doesn't lie low."
 }
 [sha:a09b4b607b62f1ee796d7e8151d4c199ee23aab6]{
 -- Page 222, Offset 9146.
 -- 「それは駄目だ、アイツはいるだけで炎をまき散らす怪物なんだぞ!?　一日だって放っておけるか！」
-"I can't allow that. He's a monster that leaves a trail of flames anywhere he goes! We can't just let him roam freely, not even for a single day!"
+"I can't allow that. He's a monster that leaves a trail of flames anywhere he goes!
+ We can't just let him do as he pleases for even %{i}one%{/i} day!"
```

Comment:
Hmm, seems kinda like a weird way to phrase it
I'd even prefer the previous but..
>...
>We can't leave him be, not even for a single day!